### ISSUE TITLE
Make Icon look like the original LeafPic

### ISSUE THREAD

**Issue by JoeJoeTV** [apcro/leafpicrevived#42]
I noticed, that the Icon does not use the entire space, like LeafPic's does.

Screenshot showing both icons:
![Screenshot-2020-12-02-10-05-19.png](https://user-images.githubusercontent.com/34209417/100851817-f9badd80-3485-11eb-99d1-9756b6e02d10.png)

I don't know if it matters, but here is some info:
- Android Version: 10
- Launcher: Lawnchair

Also, there hasn't been a commit since 6 months, so, is this still in development?

**Comment by apcro** [comment]
This is an interesting issue, as the icons I'm using are the original ones from when I first forked the project, although perhaps with changes to how icons are handled the old ones may not be completely compatible anymore.

Development is still ongoing, just nothing to be committed yet. Unfortunately the major piece of work I am doing is to deal with the breaking changes introduced in Android Q (see https://github.com/apcro/leafpicrevived/issues/28) and I've also found myself unexpectedly busy the past 9 months which has had a negative impact on my ability to devote the contiguous time these changes need.

I'll see about an interim update in the next weeks to pick up some of the other issues.
